Xiaomi/RedmiNote10S-SystemUI/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.xiaomi.redminote10s.systemui"
+        android:versionCode="1"
+        android:versionName="1.0">
+        <overlay android:targetPackage="com.android.systemui"
+                android:requiredSystemPropertyName="ro.product.vendor.device"
+                android:requiredSystemPropertyValue="+(rosemary|secret|maltose)"
```

Comment:
Matching any device called "secret" can't be good....

Reply:
ok, I will change it to matching the `ro.vendor.build.fingerprint` instead